CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.2.2
+
+**Release date:** 2024-02-01
+
+This patch release comes with various bug fixes and improvements. The kustomize
+build regression for empty directory has been fixed. The regression due to which
+the namespaced objects without a namespace specified resulted in `not found`
+error instead of `namespace not specified` has also been fixed. And the
+regression due to which Roles and ClusterRoles were reconciled over and over due
+to the normalization of Roles and ClusterRoles has also been fixed.
```

Comment:
```suggestion
This patch release comes with various bug fixes and improvements.

Reconciling empty directories and  directories without Kubernetes manifests no longer results in an error. This
regressing bug was introduced with the controller upgrade to Kustomize v5.3 and has been fixed in this patch release.

The regression due to which the namespaced objects without a namespace specified resulted in `not found`
error instead of `namespace not specified` has also been fixed. And the
regression due to which Roles and ClusterRoles were reconciled over and over due
to the normalization of Roles and ClusterRoles has also been fixed.
```